bundles/org.openhab.core.automation.module.script/src/test/java/org/openhab/core/automation/module/script/internal/ScriptEngineManagerImplTest.java

```diff
@@ -73,7 +113,7 @@ public void setup() {
         when(scriptEngineFactoryMock.getDependencyTracker()).thenReturn(scriptDependencyTrackerMock);
         when(scriptDependencyTrackerMock.getTracker(any())).thenReturn(dependencyListenerMock);
 
-        scriptEngineManager = new ScriptEngineManagerImpl(scriptExtensionManagerMock);
+        scriptEngineManager = new ScriptEngineManagerImpl(scriptExtensionManagerMock, new SafeCallerMock());
```

Comment:
Why not just use `new SafeCallerImpl(null)` instead of adding all the code for the mock?

Reply:
Yes I also saw it was in an internal package after adding my comment. It's a trade off between more code and having to maintain that and staying away from internal classes. :upside_down_face: